### PR TITLE
Use a new hub list JSON file to increase the service resiliency

### PR DESCRIPTION
**Pull request by chavoosh**

### CODE REVIEW

File: config.dist.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 GEODB_PATH = '%s/.tmp/geodb.mmdb' % BASEDIR
 
 # URL to NDN hub information
-HUBS_URL = 'http://ndnmap.arl.wustl.edu/json/geocode/'
+HUBS_URL = 'http://ndndemo.arl.wustl.edu/testbedNodes.json'
```
Comment (Pesa) [inline config.dist.py]: is this the new canonical URL for this file? it should be made available over HTTPS

Comment (chavoosh) [reply config.dist.py]: John decided to host this file on a different web server. BTW, this file comes with a new field called `ndn-up` for each hub to show whether the corresponded hub is alive.

Apparently, it cannot be served over HTTPS right now. I can ask John to take a look at it.

Comment (Pesa) [reply config.dist.py]: Yes please. I'm not a fan of the file name either... that uppercase "N"... better to stick to all lowercase letters, but I'm nitpicking now.

Comment (chavoosh) [reply config.dist.py]: I can ask John to see whether he can update the file name, too... Something like `testbed-nodes.json`

**Comment by chavoosh** [comment]
I do not think this error is related to this change... Any thoughts?

File: config.dist.py (hunk 1 above)
Comment (cawka) [reply config.dist.py]: John have updated the file name...

Comment (chavoosh) [reply config.dist.py]: Yeah. I asked him to make is accessible over HTTPS, too. He said he will probably do it by Monday.

I am still confused about the error... I have no clue how to resolve it. It does not even look like to be related to the change I submitted. Any thoughts?

Comment (Pesa) [reply config.dist.py]: Thanks! Still not reachable over https though

**Comment by Pesa** [comment]
It's a pre-existing error, the CI scripts are broken or something... not sure how @cawka wants to proceed...

**Comment by cawka** [comment]
Chavoosh can rebase on top of the current master.  CI error is fixed now.

**Review by Pesa — COMMENTED** [review]
This commit does more than "update hubs url", can you change the commit message?

**Comment by chavoosh** [comment]
So, technically, in this commit the new hub list JSON file is used (that includes a new field, named `ndn-up`). Accordingly, the server responds to queries by checking the `ndn-up` field to avoid connecting end-users to dead or down hubs.

**Comment by chavoosh** [comment]
Any update?

**Pull request merged**